doc/Caliptra.md

```diff
@@ -473,30 +473,30 @@ An example of when an owner must protect assets would be moving from secure mode
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
     <td>Mode selection (Boot Media Integrated and dependent selections)</td>
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
-    <td>Pauser Attribute</td>
+    <td>Pauser attribute</td>
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
     <td>JTAG debug</td>
     <td>Integrity</td>
     <td>Proficient</td>
-    <td>1. Attempt to manipulate RoT execution via JTAG to non POR flows <br>2. Attempt to retrieve device secrets via JTAG when product is field-deployed<br>3. Attempt to retrieve device secrets via JTAG when product is under development/debug</td>
+    <td>1. Attempt to manipulate RoT execution via JTAG to non POR flows <br>2. Attempt to retrieve device secrets via JTAG when product is field-deployed<br>3. Attempt to retrieve device secrets via JTAG while the product is being developed and debugged</td>
```

Comment:
I think "POR" should be defined earlier, expanded (plan-of-record), or reworded (maybe "to non-approved flows", or "to attacker-controlled flows")